src/pages/home/sidebar/ChatLinkRow.js

```diff
@@ -85,6 +92,34 @@ const ChatLinkRow = ({
                             </View>
                         )
                     }
+                    {/* For group chat with two or more participants */}
+                    {
+                        Boolean(option.groupIcons && !option.icon && option.groupIcons.length)
+                        && (
+                            <View style={styles.chatSwitcherAvatarContainerForGroup}>
+                                <View style={styles.chatSwitcherAvatar1ForGroup}>
+                                    <Image
+                                        source={{uri: option.groupIcons[0]}}
+                                        style={[styles.chatSwitcherAvatarImageForGroup]}
+                                    />
+                                </View>
+                                <View style={[styles.chatSwitcherAvatar2ForGroup, groupAvatar2DynamicStyles]}>
+                                    {
+                                        circularCount ? (
+                                            <Text style={styles.groupAvatarCircularCountText}>
+                                                {circularCount}
+                                            </Text>
+                                        ) : (
+                                            <Image
+                                                source={{uri: option.groupIcons[1]}}
+                                                style={[styles.chatSwitcherAvatarImageForGroup]}
+                                            />
+                                        )
+                                    }
+                                </View>
+                            </View>
```

Comment:
This should be made into its own component.

Reply:
Fixed. Please check.